go/vt/vttablet/onlineddl/executor.go

```diff
@@ -3524,46 +3524,68 @@ func (e *Executor) SubmitMigration(
 	}
 
 	if onlineDDL.StrategySetting().IsSingleton() || onlineDDL.StrategySetting().IsSingletonContext() {
-		e.migrationMutex.Lock()
-		defer e.migrationMutex.Unlock()
+		// we need two wrap some logic within a mutex, so as to make sure we can't submit the migration whilst
```

Comment:
Most of the change here is indentation :) try reviewing the change in https://github.com/vitessio/vitess/pull/9704/files?w=1